apps/src/templates/sharing.html.ejs

```diff
@@ -39,7 +39,7 @@
       <button class="saved-to-gallery" disabled>
         <%= msg.savedToGallery() %>
       </button>
-    <% } else if (options.saveToGalleryUrl && !options.disableSaveToGallery) { %>
+    <% } else if ((options.saveToProjectGallery || options.saveToGalleryUrl) && !options.disableSaveToGallery) { %>
```

Comment:
This ejs will be turning into React soon! https://docs.google.com/document/d/1zZAAWR-QrazI2GMQIRkJ-eK53e1WDGHCQq0fBzZGSos/edit#heading=h.c7tcjbjblqqf

<img width="623" alt="screen shot 2017-04-28 at 12 46 46 pm" src="https://cloud.githubusercontent.com/assets/8001765/25544760/ca42f4a6-2c10-11e7-8e08-90b52ec4a6fb.png">